spectrocloud/data_source_appliances.go

```diff
@@ -0,0 +1,73 @@
+package spectrocloud
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spectrocloud/hapi/models"
+	"github.com/spectrocloud/terraform-provider-spectrocloud/pkg/client"
+	"reflect"
+)
+
+type MyMap map[string]string
```

Comment:
Is `MyMap` being used? If not, let's remove it. If yes, can we use a better name please?

Reply:
No, it's not I meant to create a unit test and forgot to delete it.